src/components/MessageIndicator/MessageIndicator.scss

```diff
@@ -15,6 +15,6 @@ $indicator-position: rem(-3px);
   width: $indicator-size;
   height: $indicator-size;
   border-radius: 100%;
-  background-color: var(--p-icon-highlight, color('teal'));
-  border: solid rem(2px) var(--p-background, var(--top-bar-background));
```

Comment:
@alex-page I'm thinking about adding these back to this PR, they might be used by plus or other teams but we should be able to remove them in v7. What do you think?